src/js/util.js

```diff
@@ -218,6 +218,17 @@ formdesigner.util = (function(){
         throw new Error("Unable to copy obj! Its type isn't supported.");
     }
     that.clone = clone;
+    
+    function invert(obj) {   
```

Comment:
ha, i clicked a link from slack and didn't realize this is an old PR. nevermind, Sravan :)